src/Resources/assets/controllers/datetime_controller.js

```diff
@@ -4,8 +4,10 @@ import easepickStyle from '!!raw-loader!@easepick/bundle/dist/index.css'
 
 export default class extends Controller {
     static values = {
-        lang: String
+        lang: String,
```

Comment:
I feel like this should not be an input value but use the native HTML `lang` attribute to control it. Because that is exactly for what it is for: Give a language hint for an element. With the hint, screenreaders and other parsers use the element in the correct language.

Is there a reason we can't use the native attribute here?

Reply:
We set the controller on the Twig block "attributes". We could check there for lang attribute and set it too.
But nothing about the markup is handled within our controller. `easypick` want that parameter to return the correct label. Setting a `lang` attribute is up to the developer on the form input if desired.